0003-configuration-management-extension/README.md

```diff
@@ -0,0 +1,75 @@
+---
+name: Configuration Management Extension
+shortname: 3/CMX
+status: raw
+editor: Dmitry Sergeev <dsergeev@cybervisiontech.com>
+---
+
+## Introduction
+
+The Configuration Management Extension is a [Kaa Protocol](/0002-kaa-protocol/README.md) extension.
+
+It is intended to manage endpoint configuration distribution.
+
+## Requirements and constraints
+### Problems and possible solutions
+
+1. _Multiple configurations available for client._ It is possible that there will be more than one available configurations for endpoint (e.g.: endpoint was offline while new configurations were added).
+   Solution:
+   - Only latest configuration should be sent to an endpoint to reduce amount of traffic.
+_Note:_ Approach for delivering firmware updates over the air should be different.
+
+2. _The server should know if a configuration has been delivered._ 
+
+   Solutions:
+   - Send PUBLISH message from client to "/status" topic. See below sections for details.
+
+## Use cases
+
+### UC1
+Configuration delivery by request. The endpoint should be able to receive latest configuration from CMX by request.
+
+### UC2
+Configuration delivery as a reaction on EP lifecycle event. The endpoint should receive latest configuration when it connects to a server if this configuration hadn't applied yet.
```

Comment:
Define `EP lifecycle event`. Also, `EP` was not defined before.

Casting @Slash32 in, as he should know more use cases.